papermill/tests/test_parameterize.py

```diff
@@ -72,3 +72,52 @@ def test_repeated_run_no_parameters_tag(self):
 
         test_nb = parameterize_notebook(test_nb, {'msg': 'Hello'})
         self.assertEqual(self.count_nb_injected_parameter_cells(test_nb), 1)
+
+
+class TestPathParameterizing(unittest.TestCase):
+    def test_plain_text_path_with_empty_parameters_object(self):
+        self.assertEqual(parameterize_path("foo/bar", {}), "foo/bar")
+
+    def test_plain_text_path_with_none_parameters(self):
+        self.assertEqual(parameterize_path("foo/bar", None), "foo/bar")
+
+    def test_plain_text_path_with_unused_parameters(self):
+        self.assertEqual(parameterize_path("foo/bar", {"baz": "quux"}), "foo/bar")
+
+    def test_path_with_single_parameter(self):
+        self.assertEqual(parameterize_path("foo/bar/{baz}", {"baz": "quux"}), "foo/bar/quux")
+
+    def test_path_with_boolean_parameter(self):
+        self.assertEqual(parameterize_path("foo/bar/{baz}", {"baz": False}), "foo/bar/False")
+
+    def test_path_with_none_parameter(self):
+        self.assertEqual(parameterize_path("foo/bar/{baz}", {"baz": None}), "foo/bar/None")
+
+    def test_path_with_numeric_parameter(self):
+        self.assertEqual(parameterize_path("foo/bar/{baz}", {"baz": 42}), "foo/bar/42")
+
+    def test_path_with_numeric_format_string(self):
+        self.assertEqual(parameterize_path("foo/bar/{baz:03d}", {"baz": 42}), "foo/bar/042")
+
+    def test_path_with_float_format_string(self):
+        self.assertEqual(parameterize_path("foo/bar/{baz:.03f}", {"baz": 0.3}), "foo/bar/0.300")
+
```

Comment:
We should maybe test list and dict inputs, since those are common parameter types.